src/firefly/java/edu/caltech/ipac/visualize/plot/PixelValue.java

```diff
@@ -180,13 +180,12 @@ static public double pixelVal(RandomAccessFile fits_file, int x, int y,
     long data_offset   = header.getDataOffset();
 
 
-	if ((naxis == 2) || (naxis3 == 1))
-	{
+    if ((naxis==2 || naxis3==1) && plane_number<1) {
 	    plane_offset = 0;
 	}
-	else
-	{
-	    plane_offset = plane_number - 1;
+	else  {
+	    plane_offset = plane_number;
+//		plane_offset = plane_number - 1;
 	}
 
```

Comment:
I don't think this block is needed.  The plane_number can be used directly in calculating offset.   I also modified codes to take  out the naxis3=1 assignment and usage.

Reply:
oops, missed you comment, I will look.